Parity/src/QwHelicityPattern.cc

```diff
@@ -765,7 +767,8 @@ void  QwHelicityPattern::AccumulateRunningBurstSum()
   }
 
   // Calculate asymmetry over this entire burst
-  fBurstAsymmetry.Ratio(fBurstDifference, fBurstYield);
+  // Update: At end of burst calculate the asymmetry eventwise now, instead of burstwise
+  //fBurstAsymmetry.Ratio(fBurstDifference, fBurstYield);
   // Accumulate this burst asymmetry
   fRunningBurstAsymmetry.AccumulateRunningSum(fBurstAsymmetry);
```

Comment:
Maybe this line should be moved inside the block after the test on fEnableRunningSum.